### PR TITLE
i18n(zh-CN): Add supabase.mdx

### PR DESCRIPTION
**Pull request by gxy5202**
Add supabase.mdx

#### Description (required)
add supabase.mdx

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by liruifengv** [comment]
There is something in the backend that is not marked as i18nready: true. check if it needs to be translated? @Yan-Thomas

File: src/content/docs/zh-cn/guides/backend/supabase.mdx [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+- package.json
+</FileTree>
+
+## 使用 Supabase 添加身份验证
+
+Supabase 提供了开箱即用的身份验证功能。它支持电子邮件（或密码身份）验证以及如GitHub, Google等多个提供商的 OAuth 身份验证。
+
+### 前期准备
+
+- [一个已经初始化 Supabase 的 Astro 项目](#initializing-supabase-in-astro).
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/backend/supabase.mdx:115]:
```suggestion
- [一个已经初始化 Supabase 的 Astro 项目](#在-astro-中初始化-supabase)。
```

File: src/content/docs/zh-cn/guides/backend/supabase.mdx [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+`register.astro` 包含一个用于创建新用户的表单，该表单接受电子邮箱和密码信息并将向 `/api/auth/register` 端点发送 `POST` 请求。
+
+```astro title="src/pages/register.astro"
+---
+import Layout from "../layouts/Layout.astro";
+---
+
+<Layout title="Register">
+  <h1>Register</h1>
+  <p>Already have an account? <a href="/signin">Sign in</a></p>
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/backend/supabase.mdx:257]: We tend to translate the English in the code into Chinese as well.

File: src/content/docs/zh-cn/guides/backend/supabase.mdx [hunk 3]
```diff
@@ -0,0 +1,503 @@
...
+    - dashboard.astro
+  - env.d.ts
+- .env
+- astro.config.mjs
+- package.json
+</FileTree>
+
+## 社区资源
+
+- [Getting into the holiday spirit with Astro, React, and Supabase](https://www.aleksandra.codes/astro-supabase)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/backend/supabase.mdx:502]: This needs to be translated as well.

**Review by liruifengv — CHANGES_REQUESTED** [review]
Thanks for your contribution. I've submitted some suggestions, and waiting for Yan's confirmation.

File: src/content/docs/zh-cn/guides/backend/supabase.mdx [hunk 4]
```diff
@@ -0,0 +1,503 @@
...
+
+<Layout title="Sign in">
+  <h1>Sign in</h1>
+  <p>New here? <a href="/register">Create an account</a></p>
+  <form action="/api/auth/signin" method="post">
+    <label for="email" for="email">Email</label>
+    <input type="email" name="email" id="email" />
+    <label for="password">Password</label>
+    <input type="password" name="password" id="password" />
+    <button type="submit">Login</button>
```
Comment (huyikai) [inline src/content/docs/zh-cn/guides/backend/supabase.mdx:292]:
```suggestion
<Layout title="登录">
  <h1>登录</h1>
  <p>New here? <a href="/register">创建帐户</a></p>
  <form action="/api/auth/signin" method="post">
    <label for="email" for="email">电子邮件</label>
    <input type="email" name="email" id="email" />
    <label for="password">密码</label>
    <input type="password" name="password" id="password" />
    <button type="submit">登录</button>
```

Comment (gxy5202) [reply src/content/docs/zh-cn/guides/backend/supabase.mdx:292]: thanks

File: src/content/docs/zh-cn/guides/backend/supabase.mdx (hunk 3 above)
Comment (gxy5202) [reply src/content/docs/zh-cn/guides/backend/supabase.mdx:502]: thanks

**Comment by gxy5202** [comment]
Thank you guys, I pushed a new commit to solve these issues.
@liruifengv @huyikai

**Comment by liruifengv** [comment]
> Thank you guys, I pushed a new commit to solve these issues. @liruifengv @huyikai

Thanks, please resolve conversations abolve.

**Comment by liruifengv** [comment]
> There is something in the backend that is not marked as `i18nready: true`. check if it needs to be translated? @Yan-Thomas

cc @Yan-Thomas @sarah11918

**Comment by gxy5202** [comment]
> > Thank you guys, I pushed a new commit to solve these issues. @liruifengv @huyikai
> 
> Thanks, please resolve conversations abolve.

Yes, i just did it, thanks again~

**Comment by Yan-Thomas** [comment]
> There is something in the backend that is not marked as i18nready: true. check if it needs to be translated?

It should be good to be translated, thanks for checking!

**Review by liruifengv — APPROVED** [review]
So, I can write LGTM!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Good job, and welcome to Team Docs @gxy5202!

**Pull request merged**